### PR TITLE
Add the Localization & Globalization help topic

### PR DESCRIPTION
**Pull request by RomanTsukanov**

### CODE REVIEW

**Review by tsv2013 — APPROVED** [review]

File: docs/localization.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+  - [Enable Localization and Switch Between Locales](#enable-localization-and-switch-between-locales)
+  - [Override Individual Translations](#override-individual-translations)
+- [Localize Survey Contents](#localize-survey-contents)
+  - [Localize `choicesByUrl`](#localize-choicesbyurl)
+
+## Localize UI Elements
+
+### Available Languages
+
+Survey UI is translated into over 50 languages. We ship translated strings as [dictionary files](https://github.com/surveyjs/survey-library/tree/master/src/localization). They are supported by the community and may be incomplete. Feel free to add missing translations to existing dictionaries or create new dictionaries for other languages. You can use English as a base dictionary: copy the file, replace English translations in it, and submit a pull request with the resulting file to the [survey-library](https://github.com/surveyjs/survey-library) repository.
```
Comment (dmitry-eliseev-devexpress) [inline docs/localization.md:16]:
```suggestion
Survey UI is translated into over 50 languages. We ship translated strings as [dictionary files](https://github.com/surveyjs/survey-library/tree/master/src/localization). They are supported by the community and may be incomplete. Feel free to add missing translations to existing dictionaries or create new dictionaries for other languages. You can use English as a base dictionary: copy the file, replace English strings with your translations in it, and submit a pull request with the resulting file to the [survey-library](https://github.com/surveyjs/survey-library) repository.
```

File: docs/localization.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+
+### Enable Localization and Switch Between Locales
+
+The localization engine that works with dictionaries is available as a separate scrips/module. Reference this script in the `<head>` tag of your page or import this module in the component that renders your survey:
```
Comment (dmitry-eliseev-devexpress) [inline docs/localization.md:20]: I believe this is a typo.

```suggestion
The localization engine that works with dictionaries is available as a separate script/module. Reference this script in the `<head>` tag of your page or import this module into the component that renders your survey:
```

File: docs/localization.md [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+
+```html
+<script src="https://unpkg.com/survey-core/survey.i18n.min.js"></script>
+```
+
+```js
+import "survey-core/survey.i18n";
+```
+
+The default language for UI elements is English. To use another language, set the Survey's [`locale`](https://surveyjs.io/Documentation/Library?id=surveymodel#locale) property. For example, the following code translates the survey UI to French:
```
Comment (dmitry-eliseev-devexpress) [inline docs/localization.md:30]:
```suggestion
The default language for UI elements is English. To select another language, use the Survey's [`locale`](https://surveyjs.io/Documentation/Library?id=surveymodel#locale) property. For example, the following code translates the survey UI to French:
```

File: docs/localization.md [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+  completeText: "Send"
+};
+
+surveyLocalization.locales["customlocale"] = customLocaleStrings;
+// ...
+// Activate the custom locale
+survey.locale = "customlocale";
+```
+
+If any translation strings are missing from your custom locale, they will be taken from the default English locale. You can specify the `defaultLocale` property to use another locale as default:
```
Comment (dmitry-eliseev-devexpress) [inline docs/localization.md:72]:
```suggestion
If any translation strings are missing in your custom locale, they will be taken from the default English locale. You can specify the `defaultLocale` property to use another locale as default:
```

File: docs/localization.md [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+
+```js
+surveyLocalization.defaultLocale = "fr";
+```
+
+[View example](https://surveyjs.io/Examples/Library/survey-localization)
+
+## Localize Survey Contents
+
+You can localize questions, choices, columns, rows, and other survey texts right in the survey JSON definition. Normally, properties that specify survey texts accept string values, as the [`title`](/Documentation/Library?id=Question#title) property in the code below:
```
Comment (dmitry-eliseev-devexpress) [inline docs/localization.md:82]:
```suggestion
You can localize questions, choices, columns, rows, and other survey texts right in the survey JSON definition. Normally, properties that specify survey texts accept string values, like the [`title`](/Documentation/Library?id=Question#title) property in the code below:
```

File: docs/localization.md [hunk 6]
```diff
@@ -0,0 +1,148 @@
...
+    "title": {
+      "default": "Enter your first name",
+      "de": "Geben Sie Ihren Vornamen ein",
+      "fr": "Entrez votre prénom"
+    }
+  }]
+};
+```
+
+You do not have to specify translations for all locales. If a translation is missing from the current locale, the survey takes it from the `default` field.
```
Comment (dmitry-eliseev-devexpress) [inline docs/localization.md:110]:
```suggestion
You do not have to specify translations for all locales. If a translation is missing for the current locale, the survey takes the translation from the `default` field.
```

**Review by dmitry-eliseev-devexpress — APPROVED** [review]

File: docs/localization.md [hunk 7]
```diff
@@ -0,0 +1,148 @@
...
+
+```html
+<script src="https://unpkg.com/survey-core/survey.i18n.min.js"></script>
+```
+
+```js
+import "survey-core/survey.i18n";
+```
+
+The default language for UI elements is English. To select another language, use the Survey's [`locale`](https://surveyjs.io/Documentation/Library?id=surveymodel#locale) property. For example, the following code translates the survey UI to French:
```
Comment (annnke) [inline docs/localization.md:30]:
```suggestion
The default language for UI elements is English. To select another language, use the survey's [`locale`](https://surveyjs.io/Documentation/Library?id=surveymodel#locale) property. For example, the following code translates the survey UI to French:
```

**Review by annnke — APPROVED** [review]

**Pull request merged**